fakturogen-web/src/main/resources/application.properties

```diff
@@ -1,3 +1,18 @@
+
+#spring.datasource.url=jdbc:h2:~/h2database/fakturogen2;DB_CLOSE_ON_EXIT=FALSE;DB_CLOSE_DELAY=-1;AUTO_SERVER=TRUE;FILE_LOCK=SOCKET;
```

Comment:
I think that line 1 to 15 we can delete, as these are only comments, and the same is in next lines